pandas/tests/indexing/test_iloc.py

```diff
@@ -591,3 +591,12 @@ def test_iloc_empty_list_indexer_is_ok(self):
         tm.assert_frame_equal(df.iloc[[]], df.iloc[:0, :],
                               check_index_type=True,
                               check_column_type=True)
+
+    def test_loc_identity_slice_returns_new_object(self):
```

Comment:
just rename to ``test_identity_slice_returns_new_object`` (and same for one in the ``test_loc`` file